chips/earlgrey/src/chip.rs

```diff
@@ -123,6 +123,7 @@ impl<'a, I: InterruptService<()> + 'a> EarlGrey<'a, I> {
                         None,
                     );
                 }
+                interrupts::RVTIMERTIMEREXPIRED0_0 => self.timer.service_interrupt(),
```

Comment:
`timer.service_interrupt()` is now called both here and in service_pending_interrupts, which seems wrong